easybuild/easyconfigs/n/numexpr/numexpr-2.8.7-foss-2023a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'numexpr'
+version = '2.8.7'
+
+homepage = 'https://numexpr.readthedocs.io/en/latest/'
+description = """The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
+ It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
+ code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
+ compiler at runtime."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/pydata/numexpr/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['df5131349ec1e4b080a3e2df7abbb7bce20d023adff633d259e1ce1549a60ed1']
+
+local_gfbf_ver = '2023a'
+
+dependencies = [
+    ('Python',       '3.11.3'),
+    ('SciPy-bundle', '2023.07', '', ('gfbf', local_gfbf_ver)),
```

Comment:
Why not use `gfbf` as the toolchain to avoid this?

Reply:
I don't think it should even be necessary to include the toolchain as is, EasyBuild should be able to pick up gfbf as a subtoolchain of foss. If you need to explicitly set the toolchain for a dependency it typically means the software is using the wrong toolchain (as it's almost certainly not going to work for a hierarchical module naming scheme).

Having said that, if `numexpr` does not require MPI, then it shouldn't need foss, and should use `gfbf` anyway

Reply:
Yeah, I can't remember why exactly I added it to begin with, I think I was probably misdiagnosing an issue somewhere. Right now I don't have time to rebuild and check that it still works, and this one has no special dependencies so it's a good candidate to split off and review separately.